spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -43,22 +43,12 @@ To get started, add this dependency to your project:
 </dependency>
 ----
 
-Use CDI to inject a `ManagedExecutor` or `ThreadContext` service:
-
-[source,java]
-----
-public class MyBean {
-    @Inject @ManagedExecutorConfig(propagated = ThreadContext.APPLICATION, cleared = ThreadContext.ALL_REMAINING)
-    ManagedExecutor executor;
-----
-
-Or you can create one using a builder:
+Use the fluent builder API to construct a `ManagedExecutor`:
 
 [source,java]
 ----
     ManagedExecutor executor = ManagedExecutor.builder()
                        .propagated(ThreadContext.APPLICATION, ThreadContext.CDI)
-                       .cleared(ThreadContext.ALL_REMAINING)
```

Comment:
Accidental removal?

Reply:
Yes, good catch. I'll get this line added back.